src/js/load-pyodide.js

```diff
@@ -14,11 +14,17 @@ export async function initializePackageIndex(indexURL) {
     fs.readFile(`${indexURL}packages.json`, (err, data) => {
       if (err) throw err;
       let response = JSON.parse(data);
-      Module.packages = response;
+      Module.packages = response["packages"];
     });
   } else {
     let response = await fetch(`${indexURL}packages.json`);
-    Module.packages = await response.json();
+    Module.packages = (await response.json())?.packages;
```

Comment:
The logic here seems a bit messy to me (though it isn't really the fault of this PR). I think the following would be better:
```
let package_json;
if(in_node){
    const fsPromises = await import("fs/promises");
	const package_string = await fsPromises.readFile(`${indexURL}packages.json`);
	package_json = JSON.parse(package_string);
} else {
    let response = await fetch(`${indexURL}packages.json`);
    package_json = await response.json();
}
if (!package_json.packages) {
  throw new Error(
    "Loaded packages.json does not contain the expected key 'packages'."
  );
}
Module.packages = package_json.packages;
Module._import_name_to_package_name = new Map();
for (let name of Object.keys(Module.packages)) {
  for (let import_name of Module.packages[name].imports) {
    Module._import_name_to_package_name.set(import_name, name);
  }
}
```

Reply:
Also, ideally we could define `IN_NODE` somewhere so that we could say `if(IN_NODE)` which would be more readable.